webapp/src/ts/services/tasks-for-contact.service.ts

```diff
@@ -35,18 +39,19 @@ export class TasksForContactService {
         }
 
         // must be either a person type
-        if (type.person) {
+        if (type?.person) {
           return true;
         }
 
         // ... or a leaf place type
         return this.contactTypesService
           .getAll()
           .then(types => {
-            const hasChild = types.some(t => !t.person && t.parents && t.parents.includes(type.id));
+            const hasChild = types.some(t => !t.person && t.parents && t.parents.includes(type?.id));
             return !hasChild;
           });
-      });
+      })
+      .catch(err => console.error('Error finding out if tasks are enabled', err));
```

Comment:
Not sure if we should be catching these. 
Not catching them should mean that a feedback doc is created automatically, which means that the error will be uploaded and be available for us to see. 
Same goes for the other `catch` that's added in this PR.

Reply:
Guess that explains why errors were not being caught. Thanks for the context. I have removed the catch blocks.